pwiz_tools/Skyline/SettingsUI/EditStaticModDlg.resx

```diff
@@ -802,7 +802,83 @@
     <value>$this</value>
   </data>
   <data name="&gt;&gt;comboMod.ZOrder" xml:space="preserve">
-    <value>1</value>
+    <value>2</value>
+  </data>
+  <metadata name="imageList1.TrayLocation" type="System.Drawing.Point, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">

```

Comment:
What is the image list for here?

Reply:
This was the only way I could figure out how to set the Transparent Color on the linked peptide button image. Was there another way I could have done this?

Reply:
Is the linked peptide image used? Or is it currently invisible for future use? I didn't see this image list used in this dialog.

Reply:
Oops. I thought we were talking about EditPepModsDlg. I'll delete this imageList from this dialog.